doc/whats_new/v1.1.rst

```diff
@@ -202,6 +202,20 @@ Changelog
   `fit` instead of `__init__`.
   :pr:`21567` by :user:`Maggie Chege <MaggieChege>`.
 
+- |Enhancement| :class:`decomposition.TruncatedSVD` exposes the parameter
+ `n_oversamples` and `power_iteration_normalizer` to tune
+  :func:`sklearn.decomposition.randomized_svd` and
+  get accurate results when the number of features is large, the rank of
+  the matrix is high, or other features of the matrix make low rank
+  approximation difficult.
+  :pr:`21705` by :user:`Jay S. Stanley III <stanleyjs>`.
+
+  - |Enhancement| :class:`decomposition.TruncatedSVD` exposes the parameter
```

Comment:
I think this is for `PCA`?

```suggestion
- |Enhancement| :class:`decomposition.PCA` exposes the parameter
```

(Also to remove the whitespace before `- |Enhancement|`)

Reply:
Thanks. I obviously need to pay more attention to detail.